src/hssm/likelihoods/analytical.py

```diff
@@ -262,15 +241,17 @@ def logp_ddm(
     z_flipped = pt.switch(flip, 1 - z, z)  # transform z if x is upper-bound response
     rt = rt - t
 
-    p = pt.maximum(ftt01w(rt, a, z_flipped, err, k_terms), pt.exp(LOGP_LB))
+    negative_rt = rt <= epsilon
```

Comment:
Ok reflecting on this a bit, the logic that we want should probably look something like:

- flag all rts lower than epsilon
- go through with ftt01w
- then set all flagged rts to `LOGB_LB`

This should actually cut the gradient for problematic rts.
Potentially we put this as a `logp_ddm_2` and compare results / gradients.
Alternatively, if any `rt` breaches epsilon, directly send logp to -infty (this is probably not preferable).